src/app/admin/users/PageUserCreate.tsx

```diff
@@ -1,24 +1,135 @@
+    onError: (error: any) => {
+      const { description } = error?.response?.data || {};
+      toastError({
+        title: 'Creation failed',
+        description,
+      });
+    },
+    onSuccess: () => {
+      toastSuccess({
+        title: 'User created with success',
+      });
+      history.push('/admin/users');
+    },
+  });
+
+  const languages = [{ label: 'English', value: 'en' }];
+
+  const authorities = Object.values(AUTHORITIES).map((value) => ({ value }));
+
+  const submitCreateUser = async (values) => {
+    const newUser = {
+      ...values,
+    };
+    console.log(newUser);
+    await createUser(newUser);
+  };
+
   return (
     <Page containerSize="md" isFocusMode>
-      <PageTopBar showBack onBack={() => history.goBack()}>
-        <Heading size="sm">PageUserCreate Component</Heading>
-      </PageTopBar>
-      <PageContent>...</PageContent>
-      <PageBottomBar>
-        <ButtonGroup justifyContent="space-between">
-          <Button onClick={() => history.goBack()}>Cancel</Button>
-          <Button colorScheme="brand">Create User</Button>
-        </ButtonGroup>
-      </PageBottomBar>
+      <Formiz
+        id="create-user-form"
+        onValidSubmit={submitCreateUser}
+        connect={createUserForm}
+      >
+        <form noValidate onSubmit={createUserForm.submit}>
+          <PageTopBar showBack onBack={() => history.goBack()}>
+            <Heading size="md">Create a new user</Heading>
+          </PageTopBar>
+          <PageContent>
+            <Stack
+              direction="column"
+              bg="white"
+              p="6"
+              borderRadius="lg"
+              spacing="6"
+              shadow="md"
+            >
+              <FieldInput
+                name="login"
+                label="Login"
+                required="This field is required"
+              />
+              <Stack direction={{ base: 'column', sm: 'row' }} spacing="6">
+                <FieldInput
+                  name="firstName"
+                  label="First Name"
+                  required="This field is required"
+                />
+                <FieldInput
+                  name="lastName"
+                  label="Last Name"
+                  required="This field is required"
+                />
+              </Stack>
+              <FieldInput
+                name="email"
+                label="Email"
+                required="This field is required"
+              />
+              <FieldSelect
+                name="langKey"
+                label="Language"
+                options={languages}
+                defaultValue={'en'}
+              />
+              <FieldGroupCheckbox
+                name="authorities"
+                label="Authorities"
+                options={authorities}
+                isRequired
+                validations={[
+                  {
+                    rule: (value) => value?.length >= 1,
```

Comment:
use `isMinLenght(1)` (https://formiz-react.com/docs/validations/rules#isminlengthmin) instead of custom validation ;)